ash/src/vk/extensions.rs

```diff
@@ -17672,6 +17672,20 @@ impl ExtPipelinePropertiesFn {
     pub const SPEC_VERSION: u32 = 1u32;
 }
 #[allow(non_camel_case_types)]
+#[doc = "Implemented for all types that can be passed as argument to `p_pipeline_properties` in [`PFN_vkGetPipelinePropertiesEXT`]"]
```

Comment:
Perhaps the docs should link to the high-level wrapper, but I'm not sure if we should assume the existence of `crate::extensions::...::PipelineProperties::get_pipeline_properties`] within the generator.